### ISSUE TITLE
customize message

### ISSUE THREAD

**Issue by Titintgn** [7orlum/SmtpTelegramGateway#7]
Good morning, it would be great to be able to choose to have only the content of the email delivered without the address or subject, to be able to personalize the message or and line to standar text and message content whitout adress or subject. Thank you.

**Comment by mandragor59** [comment]
I agree with that. I would also think it would be cool if HTML emails were supported, like in Telegram's Temp-Email-bot. All my emails are transmitted as text only. Or am I doing something wrong or is this a functionality of the bot ?

**Comment by 7orlum** [comment]
This program was conceived as a way to forward monitoring messages from programs that only support sending to email. I'm curious about your use case. What do you use this program for? This will help me understand how to best implement your request.

**Comment by mandragor59** [comment]
I mainly use your tool to forward high-priority emails to myself and my family when we are traveling and do not have direct access to our email server. Of course, this also applies to monitoring emails from our home automation system, which also include graphics showing consumption and energy generation data from our solar systems etc.

**Comment by 7orlum** [comment]
I think the simplest and most effective option in this case would be for the program to send emails to Telegram as an “eml” file. This format preserves the original email in its entirety, including attachments. From Telegram on Android, it can be opened in Gmail or Chrome.

**Comment by mandragor59** [comment]
Nice suggestion. Can you also tell me how that would work with the special programs that can only send emails?  ...... And it seems to contradict your previous description of your project's goal a little.

**Comment by 7orlum** [comment]
You won't need to change anything in the programs that send emails. Only the format of Telegram messages will change: instead of a text messages, the program will send an .eml files to your Telegram.
[What are .eml files](https://www.indeed.com/career-advice/career-development/eml)

**Comment by mandragor59** [comment]
I am well aware of what .eml files are. But how exactly can i change the transmission as an “.eml” in your program for Telegram if i must not change the behavior of the sending app's ? Please explain it to me in  detail how this will work.

**Comment by 7orlum** [comment]
I need time to make this happen.

**Comment by mandragor59** [comment]
Ahh... now I understand. This feature doesn't exist yet, right? Well then, take your time. However, it would be nicer to have a feature like the one in Telegram's Temp Email bot. Your program has the incredible advantage of being able to handle many email addresses and Telegram IDs. So keep at it and good luck.
